src/js/tabs/osd.js

```diff
@@ -918,505 +1252,242 @@ OSD.loadDisplayFields = function() {
             text: 'osdTextElementRtcDateTime',
             desc: 'osdDescElementRtcDateTime',
             default_position: -1,
-            draw_order: 360,
```

Comment:
ignore all these changes, as it is just moved/reordered and `git` sees them as changes.